README.md

```diff
@@ -558,6 +566,17 @@ Pebble.addEventListener('webviewclosed', function(e) {
 
 ### `Clay([Array] config, [function] customFn, [object] options)`
 
+#### Properties
+
+| Property | Type | Description |
+|----------|------|-------------|
+| `.config` | Array | Reference to the config passed to the constructor and used for generating the page. **WARNING** this is a direct reference, not a copy of the config so any modification you make to it, will be reflected on the original as well |
```

Comment:
not while it is open, no. This is explaining what is happening here: https://github.com/pebble/clay/pull/30/files#diff-04c6e90faac2675aa89e2176d2eec7d8R543